modules/flatgeobuf/package.json

```diff
@@ -33,7 +33,7 @@
   },
   "dependencies": {
     "@loaders.gl/loader-utils": "^2.1.1",
-    "flatgeobuf": "^3.1.0"
+    "flatgeobuf": "^3.5.0"
   },
   "devDependencies": {
     "esm": "^3.2.25"
```

Comment:
It should be possible to use uncompiled ES6 files from `flatgeobuf/lib` with `esm`, but I'm not sure how to wire that up.

Reply:
isn't `esm` an equivalent of `reify` which we currently use. It enables ES6 imports in Node.js.

I don't think it has any effect in the browser.

Users on older browsers will not be able to use this module if the source is not transpiled. 

> to support geojson and binary parsing we need to take from two overlapping dist bundles.

Why is that ?